From ae2e4d2719b745cd7aefc58f5bc42427aad73e27 Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Fri, 13 Jun 2014 14:36:13 -0400
Subject: [block] blk-mq: remove stale comment for blk_mq_complete_request()

Message-id: <1402670238-13916-86-git-send-email-snitzer@redhat.com>
Patchwork-id: 83821
O-Subject: [RHEL7.1 PATCH 085/150] blk-mq: remove stale comment for blk_mq_complete_request()
Bugzilla: 1105204
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

BZ: 1105204

Upstream commit 7738dac4f697ffbd0ed4c4aeb69a714ef9d876da
Author: Jens Axboe <axboe@fb.com>
Date:   Wed May 28 08:06:34 2014 -0600

    blk-mq: remove stale comment for blk_mq_complete_request()

    It works for both IPI and local completions as of commit
    95f096849932.

    Signed-off-by: Jens Axboe <axboe@fb.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/linux/blk-mq.h b/include/linux/blk-mq.h
index 87ce51b..0bee6b8 100644
--- a/include/linux/blk-mq.h
+++ b/include/linux/blk-mq.h
@@ -172,11 +172,6 @@ void blk_mq_end_io(struct request *rq, int error);
 void __blk_mq_end_io(struct request *rq, int error);
 
 void blk_mq_requeue_request(struct request *rq);
-
-/*
- * Complete request through potential IPI for right placement. Driver must
- * have defined a mq_ops->complete() hook for this.
- */
 void blk_mq_complete_request(struct request *rq);
 
 void blk_mq_stop_hw_queue(struct blk_mq_hw_ctx *hctx);
-- 
1.7.1